src/main/java/uk/org/okapibarcode/output/SvgRenderer.java

```diff
@@ -79,11 +82,27 @@ public class SvgRenderer implements SymbolRenderer {
      *        standalone SVG documents, {@code false} for SVG content embedded directly in HTML documents)
      */
     public SvgRenderer(OutputStream out, double magnification, Color paper, Color ink, boolean xmlProlog) {
+        this(out, magnification, paper, ink, xmlProlog, 0);
+    }
+
+    /**
+     * Creates a new SVG renderer with the specified clockwise rotation.
```

Comment:
No need to mention the rotation in the method description, can be same as other constructor.